index.js

```diff
@@ -61,13 +539,14 @@ app.intent('send.message', async (conv, {target, message}) => {
 
   if (!users.length && !convs.length) {
     conv.ask(`I cannot find any user or conversation called ${target}. What's the name?`);
+    conv.contexts.set('sendmessage_getconv', 5);
```

Comment:
What this a bug in the existing code? If so what was the bug?

Reply:
Yes. After a user and conversation could not be found, the Google Assistant would immediately go to the default fallback intent rather than the <b>collect.target</b> followup intent.